### PR TITLE
Update default image quality / run formatter

### PR DESCRIPTION
**Pull request by tyler-romero**
Update default camera settings to capture higher quality images. Plus a bit of cleanup and pulling some things into a `config.h` file.

(Sorry about all the formatting changes, vscode's c++ extension auto-formatted everything).

### CODE REVIEW

File: edgelight/edgelight.ino [hunk 1]
```diff
@@ -274,35 +280,45 @@ void setup() {
   config.xclk_freq_hz = 20000000;
   config.pixel_format = PIXFORMAT_JPEG;
 
-  config.frame_size = FRAMESIZE_VGA; // other options include QVGA/CIF/VGA/SVGA/XGA/SXGA/UXGA
-  config.jpeg_quality = 20; // 20 at VGA should keep it under 16kb total in most cases
+  // Photo Quality Settings
+  config.frame_size = FRAMESIZE_UXGA; // See here for a list of options and resolutions: https://github.com/espressif/esp32-camera/blob/master/driver/include/sensor.h#L84
+  config.jpeg_quality = 10;           // lower means higher quality
```
Comment (tyler-romero) [inline edgelight/edgelight.ino:279]: These lines contain the only real (non-formatting) changes

Comment (positavi) [reply edgelight/edgelight.ino:279]: there is some perf tradeoff here.  agreed that the default higher quality feels higher quality for first use though.  curious if we get better quality by scaling down the higher res image vs starting with the low-res image from the camera.  unclear how it generates the different resolution images internally.

File: edgelight/edgelight.ino [hunk 2]
```diff
@@ -38,218 +36,226 @@ SOFTWARE.
 #include "Base64.h"
 
 #ifdef ESP32
```
Comment (positavi) [inline edgelight/edgelight.ino]: can get rid of this too.  this code is designed for a pretty specific platform and requires SPIFFS.

File: edgelight/edgelight.ino [hunk 3]
```diff
@@ -38,218 +36,226 @@ SOFTWARE.
...
-#define Y5_GPIO_NUM       21
-#define Y4_GPIO_NUM       19
-#define Y3_GPIO_NUM       18
-#define Y2_GPIO_NUM        5
-#define VSYNC_GPIO_NUM    25
-#define HREF_GPIO_NUM     23
-#define PCLK_GPIO_NUM     22
-
-#define GPIO_LED_FLASH    4
+#define PWDN_GPIO_NUM 32
```
Comment (positavi) [inline edgelight/edgelight.ino:50]: the autoformatter has poor taste in typesetting !

File: edgelight/edgelight.ino [hunk 4]
```diff
@@ -38,218 +36,226 @@ SOFTWARE.
...
 #endif
 
-
-char groundlight_endpoint[40]="api.groundlight.ai";
+// Options to configure these settings will be surfaced when the user connects to the device's AP
+char groundlight_endpoint[40] = "api.groundlight.ai";
 char groundlight_API_key[75] = "api_yourgroundlightapikeyhere";
 char groundlight_det_name[100] = "det_xxxyourdetectoridhere";
 char groundlight_det_query[100] = "your query text here?";
 char groundlight_det_confidence[5] = "0.9";
```
Comment (positavi) [inline edgelight/edgelight.ino]: pull from the config default ?

File: edgelight/edgelight.ino [hunk 5]
```diff
@@ -38,218 +36,226 @@ SOFTWARE.
...
 char groundlight_take_action_on[6] = "YES";
 char groundlight_action_channel[10] = "SLACK";
 char slack_url[100] = "https://hooks.slack.com/services/blah/blah/blah";
-char delay_between_queries[10] = "30000";
+char delay_between_queries_ms[10] = "30000";
```
Comment (positavi) [inline edgelight/edgelight.ino:101]: pull from the config default?

**Review by positavi — CHANGES_REQUESTED** [review]
the defaults added in config aren't fully plumbed, should populate the defaults in setup

File: edgelight/edgelight.ino (hunk 3 above)
Comment (tyler-romero) [reply edgelight/edgelight.ino:50]: yeah it just bulldozed all that nice alignment!

File: edgelight/edgelight.ino (hunk 1 above)
Comment (tyler-romero) [reply edgelight/edgelight.ino:279]: Agree we should tune this a bit more in the future. The previous settings made the captured images with the m5stack timer camera quite blurry, but we could try adjusting the quality back down as long as the images still look fine to human labelers.

File: edgelight/edgelight.ino (hunk 5 above)
Comment (tyler-romero) [reply edgelight/edgelight.ino:101]: Decided the default config was a bit confusing. I just treat the values here as the defaults now.

**Pull request merged**